composer.json

```
@@ -9,7 +9,7 @@
         "cakephp/cakephp": "^5.0.1",
         "cakephp/migrations": "^4.0.0",
         "cakephp/plugin-installer": "^2.0",
-        "mobiledetect/mobiledetectlib": "^3.74"
+        "mobiledetect/mobiledetectlib": "^4.8"
```

```suggestion
        "mobiledetect/mobiledetectlib": "^4.8.03"
```
No other code change required https://github.com/serbanghita/Mobile-Detect/releases/tag/4.8.03

---

wow 🙈 well then 😁